modular_zzplurt/code/game/objects/items/storage/boxes/clothes_boxes.dm

```diff
@@ -0,0 +1,12 @@
+/obj/item/storage/box/lewd_toys
+	name = "lewd toys box"
+	desc = "Contains lewd impliments for spending time alone, or together with someone! Try to hide it better next time."
+
+/obj/item/storage/box/lewd_toys/PopulateContents()
+	new /obj/item/clothing/sextoy/dildo(src)
+	new /obj/item/clothing/sextoy/buttplug(src)
+	new /obj/item/stack/shibari_rope/full(src)
+	new /obj/item/spanking_pad(src)
+	new /obj/item/clothing/mask/ballgag(src)
+	new /obj/item/clothing/suit/straight_jacket/shackles(src)
+	new /obj/item/clothing/glasses/blindfold/kinky(src)
```

Comment:
```suggestion
/obj/item/storage/box/lewd_toys
	name = "lewd toys box"
	desc = "Contains lewd impliments for spending time alone, or together with someone! Try to hide it better next time."

/obj/item/storage/box/lewd_toys/PopulateContents()
	new /obj/item/clothing/sextoy/dildo(src)
	new /obj/item/clothing/sextoy/buttplug(src)
	new /obj/item/stack/shibari_rope/full(src)
	new /obj/item/spanking_pad(src)
	new /obj/item/clothing/mask/ballgag(src)
	new /obj/item/clothing/suit/straight_jacket/shackles(src)
	new /obj/item/clothing/glasses/blindfold/kinky(src)
	new /obj/item/storage/box/portal_fleshlight(src)
```
Since these exist now